packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/openidconnect/appidpinokta.md

```diff
@@ -0,0 +1,11 @@
+By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory. If your Identity Provider doesn't support this attribute, you can make it optional. See [Manage User Profiles](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-editor.htm).
+
+After you fill in your OpenID Connect Identity Provider's settings, add the following endpoint URIs for the generic OpenID Connect Identity Provider that you are configuring. You can find the endpoints in the well-known configuration document for the Identity Provider (for example, `https://theIdPorg.com/.well-known/openid-configuration`. 
+
+For a list of fully-tested and supported Identity Providers, see [OpenID Connect Identity Providers](/docs/guides/federate-with-oidc/configure-idp/#openid-connect-identity-providers).
```

Comment:
For OIDC, will the linked document still exist? Or is that info moving here?

Reply:
it's moving here:

For a list of fully-tested and supported Identity Providers, see [OpenID Connect Identity Providers](/docs/guides/add-an-external-idp/before-you-begin/#supported-openid-connect-identity-providers).